### PR TITLE
Feature paypal redirect

### PR DESCRIPTION
**Pull request by PatrickJamesHoban**
Updated redirect links for paypal one-time donation so that if donation is made paypal will redirect back to thank you page and if user cancels out of donation, they will be taken back to the site donate page.

image

image

I also updated the upcoming page as I noticed some of the code had been overwritten to a seemingly older version.  The update allows for the brown paper ticket link to appear on the upcoming event automatically when the user adds the ticket code to the md file.

Without code in md file...
image

This is the update in the code for the upcoming.html...
image

Here is where the user would add the code in the md file...
image

Here is what the site would look like after code is added in md file...
image

### CODE REVIEW

File: _includes/upcoming.html [hunk 1]
```diff
@@ -3,15 +3,12 @@
...
-  {% if tickets.code %}
-    <link rel="stylesheet" type="text/css" href="http://www.brownpapertickets.com/widget_v651.css" /><DIV ID="bpt_eventbody"><CENTER><BR><BR>Brown Paper Tickets Ticket Widget Loading...<BR><BR><A HREF="http://www.brownpapertickets.com/event/{{ tickets.code }}">Click Here</A> to visit the Brown Paper Tickets event page.</CENTER><BR><BR></DIV> <script src="http://www.brownpapertickets.com/eventwidget.js?event={{ tickets.code }}&nodescription=1" type="text/javascript" language="javascript"></script>
-    <script src="http://www.brownpapertickets.com/widget_v651.js?event={{ tickets.code }}" type="text/javascript" language="javascript"></script>
-  {% endif %}
-{% endfor %}
-{% for tickets in event.tickets %}
-  {% if tickets.code %}
-    <A HREF="http://www.brownpapertickets.com/event/{{ tickets.code }}"><IMG SRC="http://www.brownpapertickets.com/g/6/BPT_buy_tickets_small.png" WIDTH=108px HEIGHT=58px BORDER=0></A>
-  {% endif %}
+  <!-- PH 3/24/17 - The below needs to be nested under site.upcoming as it is now -->
```
Comment (hollabaq86) [inline _includes/upcoming.html]: these comments are visible in inspector, we may want to move to README or instructions.txt file

File: _includes/upcoming.html [hunk 2]
```diff
@@ -3,15 +3,12 @@
...
+  <!-- The link for tickets will then only appear when the actual md file includes a code. -->
+  <!-- If there is no code present in the md file, the link will NOT generate on the page. -->
+  {% for tickets in event.tickets %}
+    {% if tickets.code %}
+      <A HREF="http://www.brownpapertickets.com/event/{{ tickets.code }}"><IMG SRC="http://www.brownpapertickets.com/g/6/BPT_buy_tickets_small.png" WIDTH=108px HEIGHT=58px BORDER=0></A>
```
Comment (hollabaq86) [inline _includes/upcoming.html]: tags and attributes are in all caps... can we change back to lowercase to match all styles?

File: donate.html [hunk 3]
```diff
@@ -11,6 +11,9 @@ <h2 class="donate-title">Make a One Time Donation </h2>
         <form class="donate-button" action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
           <input type="hidden" name="cmd" value="_s-xclick">
           <input type="hidden" name="hosted_button_id" value="C69YKQAPRZM5G">
+          <!-- PH 3/24/17 - Adding the return links back to site following paypal donation -->
```
Comment (hollabaq86) [inline donate.html]: appreciate adding this comment, but it's not needed. Please remove

**Review by hollabaq86 — CHANGES_REQUESTED** [review]

**Comment by PatrickJamesHoban** [comment]
@hollabaq86 - Hey Holly, Finally got around to making those few small tweaks you requested.  I pulled the latest master, merged into the feature branch on my machine, made those three edits you asked for, I double checked everything still worked on my localhost and then pushed the feature back up again.  This pull request is now reflective of those changes and ready for merge.  Let me know if you have questions or need anything else!  ~PH

**Review by hollabaq86 — APPROVED** [review]

**Pull request merged**